src/helpers/dom/element.js

```diff
@@ -292,12 +292,17 @@ if (isClassListSupported()) {
   };
 
   _addClass = function(element, classes) {
+    if (!classes) {
+      return;
+    }
```

Comment:
I believe this check will catch only `addClass(element, undefined)` call. If you're intention (as described) is to sync the behavior of `addClass` for different browsers then you should filter `addClass(element, ["aa", undefined, "bbb"])` use case:

https://github.com/handsontable/handsontable/blob/f78bfbcc6b8818cec392b33a836dfdb5f5836477/src/helpers/dom/element.js#L239-L243

This check may still be valid, we should consider copying it the second `_addClass` as well because calling `addClass` with an undefined will throw an error

Reply:
Sure... ;) Thanks! Changed within: https://github.com/handsontable/handsontable/pull/7064/commits/34f439f1c6d96ff361aaadfe3b4c5423c7d0fdb0.